components/CoffeLogo.tsx

```diff
@@ -0,0 +1,25 @@
+import { FC } from 'react'
+
+import { useDarkMode } from '@/lib/use-dark-mode'
+
+import styles from './styles.module.css'
+
+const CoffeLogo: FC = () => {
+  const { isDarkMode } = useDarkMode()
+  return (
+    <svg
```

Comment:
Might it be read as a file? I think this code is more readable if it'll be a file

Reply:
having it own component make it easier to read and reusable everywhere in your project. As you can see there is a classname included to it changes the colour to when the screen is dark and white. also less mb